src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -67,11 +67,7 @@
 {% block ezdatetime_field %}
 {% spaceless %}
     {% if not ez_is_field_empty( content, field ) %}
-        {% if fieldSettings.useSeconds %}
-            {% set field_value = field.value.value|localizeddate('short', 'medium', null, ez_user_settings['timezone']) %}
-        {% else %}
-            {% set field_value = field.value.value|localizeddate('short', 'short', null, ez_user_settings['timezone']) %}
-        {% endif %}
+        {% set field_value = field.value.value|ez_full_datetime %}
```

Comment:
We have an issue here as you can set `display seconds` in content type. This change will make the setting not really useful in Admin interface. This is something @SylvainGuittard should take a look at.

Reply:
This topic has been discussed with @tischsoic 
The idea was to force the rendering of seconds if the field has `use seconds`.

Reply:
@SylvainGuittard we need to decide what we want to display, seconds or format chosen by the user in settings.

Reply:
@pawbuj format chosen by the user + seconds if the field has `use seconds` checked.

Reply:
@SylvainGuittard there some technical issues here I have sent you a direct message.